en/docs/quick-starts/cross-protocol-logout-sample.md

```diff
@@ -10,10 +10,16 @@ If you have your own applications, click the button below.
 
 ----
 
+(TODO: dev-portal-fragment)
 {!fragments/pickup-dispatch-saml.md!}
 
 ----
 
+**Next, register a service provider and configure authentication for it with a different protocol.**
```

Comment:
Is this a fragment? Why have we made this bold?

Reply:
no its not a fragment...it just a part that's important but gets easily missed on that page so I bolded it to make it noticeable...WDYT?

Reply:
Might be mistaken for a heading and if there is going to be a heading before or after this, it might get confusing IMO. Guess you can decide based on the content that precedes and follows

Reply:
IMO it looks neater on the page this way and could get missed if not bolded considering the images and rest of the content on the page. 

since its a sentence, it simply looks like its bolded to show that its important.